src/fable-library/SystemException.ts

```diff
@@ -0,0 +1,9 @@
+import { Exception } from "./Types.js";
+
+export declare class SystemException extends Exception {
+    constructor();
+}
+
+export declare class TimeoutException extends SystemException {
+    constructor();
+}
```

Comment:
Because we already have the F# version we probably can remove this one. It's likely that it just gets overwritten by the F# one during build.

Reply:
I tried removing it but seems to be required by `Async.ts`

Reply:
Ah, I see, thanks! Well, then the next move would be to convert Async.ts to F# :)